src/views/404.js

```diff
@@ -1,9 +1,27 @@
-import React from 'react'
+import React from 'react';
+import Logo from '../assets/VeriFactLogo.svg';
 
-export default function NotFound (props) {
-  return (
-    <div>
-      <h1>404 page</h1>
-    </div>
-  )
+export default function NotFound(props) {
+	return (
+		<div
+			style={{
+				display: 'flex',
+				justifyContent: 'center',
+				flexDirection: 'column',
+				textAlign: 'center',
+				alignItems: 'center',
+				height: '100vh'
+			}}
+		>
+			<h1 style={{ fontSize: '6rem' }}>
+				Error 404! <br />Page Not Found!
+			</h1>
+			<p style={{ fontSize: '3rem', lineHeight:'normal' }}>
+				Seems like you're lost, <br />follow this little bee to return home.
```

Comment:
I think you should remove the `<br/>`. It's better to let the browser handle line breaks so that you don't have more breaks than necessary as screen size decreases. You might set a max-width instead if that's your concern though.

Reply:
The text is nice though 😸